File: juego/carta.go
package juego

type Carta struct {
	nombre   string
	atributo string
	poder    *Puntos
}

func NewCarta(poder *Puntos, nombre string, atributo string) *Carta {
	return &Carta{poder: poder, nombre: nombre, atributo: atributo}
}

func (c *Carta) Pelear(b *Carta) {

}

func (c *Carta) String() string {
	result := c.nombre + ", Puntos: " + c.poder.String()
	return result
}

File: juego/carta.py
class Carta():
    def __init__(self, name, description):
        self.name = name
        self.description = description

class CartaMonstruo(Carta):
    def __init__(self,nombre, description, atributo, puntos):
        super().__init__(nombre, description)
        self.atributo = atributo
        self.puntos = puntos

    def __str__(self):
        return f"{self.name} ({self.atributo}): {self.description} | Poder: {self.puntos}"